.github/workflows/pull-request.yml

```
@@ -280,20 +281,25 @@ jobs:
       - uses: subosito/flutter-action@v2
         with:
-          flutter-version: "3.29.3"
+          flutter-version: ${{ env.FLUTTER_VERSION }}
           channel: "stable" # or: 'beta', 'dev' or 'master'
           architecture: x64
       - name: Copy .env.example to .env
         run: cp .env.example .env
       - name: Building for ios
         run: flutter build ios --release --no-codesign
-
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)